libs/navigation/navigation.css

```diff
@@ -36,4 +36,8 @@ header.global-navigation, header.global-navigation.feds--dark {
   .feds-promo-link:hover {
     color: #136FF6;
   }
+  .feds--full-width .feds-topnav{
+    max-width: none;
+    padding: 0 10px 0 15px;
```

Comment:
I see two issues here:
1. this doesn't work on RTL
2. we're adding different left / right padding in order to center the content, but as soon as the content is changed (e.g. a different logo or different usage of universal navigation) things will not be centered anymore. Shouldn't we handle these use-cases as well?

Reply:
@narcis-radu Regarding the above points:

1. I only removed the center-alignment behavior and did not make any changes related to RTL functionality. It should function as it did before. Is there a particular scenario you’d like to point out?
2. The change was made to enable full-width behavior specifically for Adobe Home, as they use a full-width GNAV. That’s why additional spacing was added, which is same as for current Adobe Home already. We plan to request that they include `feds--full-width` in their markup. This change is not being applied across the entire Milo codebase, but only for the standalone GNAV.

Reply:
I agree with @narcis-radu that I don't see a point in not using the same value for right and left paddings. The logo has a left and right padding of `8px`. Utilities have a left and right padding of `8px`. Why would we not have the same value for the entire navigation? This should be either `0 10px` or `0 15px`, which will reflect nicely on RTL as well.

If you leave it as is, you will not actually have the navigation centered, it will lean towards the left on LTR or the right on RTL by `5px`.

Reply:
Agreed, I’ve updated it to 0 15px to prevent any misalignment in the future, whether for different logos or varying uses of the universal navigation